packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/examples/loyalty-example/useLoyaltyPoints.tsx

```diff
@@ -0,0 +1,50 @@
+import {useEffect, useState} from 'react';
+import {serverUrl} from './LoyaltyPointsBlock';
+
+export const useLoyaltyPoints = (
+  api: any,
+  customerId: number,
+  setPointBalance: React.Dispatch<React.SetStateAction<number | null>>,
+) => {
+  const [loading, setLoading] = useState(true);
```

Comment:
I recommend initializing the loading state to `false`. We don't want to have unnecessary loading states for cases where the extension doesn't need to fetch points.